python/ray/rllib/a3c/shared_model.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from ray.rllib.a3c.policy import (
+    categorical_sample, linear,
+    normalized_columns_initializer, Policy)
+
+from ray.rllib.models.catalog import ModelCatalog
+
+
+class SharedModel(Policy):
+    def __init__(self, ob_space, ac_space, **kwargs):
+        super(SharedModel, self).__init__(ob_space, ac_space, **kwargs)
+
+    def setup_graph(self, ob_space, ac_space):
+        num_actions = ac_space.n
+        self.x = tf.placeholder(tf.float32, [None] + list(ob_space))
+        dist_class, dist_dim = ModelCatalog.get_action_dist(ac_space)
+        self._model = ModelCatalog.get_model(self.x, dist_dim)
+
+        # self.logits = linear(self._model.last_layer, num_actions, "action",
+        #                      normalized_columns_initializer(0.01))
```

Comment:
Let's remove this comment. Unless you think we may comment it back in at some point, in which case let's add a comment explaining when we would want to do that.